### ISSUE TITLE
Bitcoind shuts off after start

### ISSUE THREAD

**Issue by assafadar** [bitcoin/bitcoin#14326]
Describe the issue 
After upgrading bitcoin core from master 0.16.99  to  0.16.3 and restarting, bitcoind shuts off after few seconds 
 
What behavior did you expect?
bitcoind to work as before

What was the actual behavior (provide screenshots if the issue is GUI-related)? 
bitcoind shuts off (tracked by 'top')

 How reliably can you reproduce the issue, what are the steps to do so? 
Reliable, run bitcoind and try to call RPC methods

 What type of machine are you observing the error on (OS/CPU and disk type)? 
Ubuntu server 16.04.5

**Comment by ReneFroger** [comment]
Could you post the latest content (around 30 lines) of the log here? You can find it in `~/.bitcoin`.

**Comment by assafadar** [comment]
Hey, sorry for delayed reply. This is my debug.log for bitcoin core... 
I can see that according to this debug log that the version of bitcoind is still 0.16.99 
I switched the bitcoind in /usr/bin/ to the new version -> which worked on other servers maintaining LTC, BCH and etc.. 

This is the debug log: 

2018-07-01T07:01:15Z Bitcoin Core version v0.16.99.0-f532d52 (release build)
2018-07-01T07:01:15Z InitParameterInteraction: parameter interaction: -whitelistforcerelay=1 -> setting -whitelistrelay=1
2018-07-01T07:01:15Z Assuming ancestors of block 0000000000000000005214481d2d96f898e3d5416e43359c145944a909d242e0 have valid signatures.
2018-07-01T07:01:15Z Setting nMinimumChainWork=000000000000000000000000000000000000000000f91c579d57cad4bc5278cc
2018-07-01T07:01:15Z Using the 'sse4(1way+4way),avx2(8way)' SHA256 implementation
2018-07-01T07:01:15Z Using RdRand as an additional entropy source
2018-07-01T07:01:15Z Default data directory /home/bitcoinuser/.bitcoin
2018-07-01T07:01:15Z Using data directory /media/bitcoin/bitcoin/
2018-07-01T07:01:15Z Using config file /.bitcoin.bitcoin.conf
2018-07-01T07:01:15Z Using at most 125 automatic connections (1024 file descriptors available)
2018-07-01T07:01:15Z Using 16 MiB out of 32/2 requested for signature cache, able to store 524288 elements
2018-07-01T07:01:15Z Using 16 MiB out of 32/2 requested for script execution cache, able to store 524288 elements
2018-07-01T07:01:15Z Using 2 threads for script verification
2018-07-01T07:01:15Z scheduler thread start
2018-07-01T07:01:15Z HTTP: creating work queue of depth 16
2018-07-01T07:01:15Z No rpcpassword set - using random cookie authentication
2018-07-01T07:01:15Z Generated RPC authentication cookie /media/bitcoin/bitcoin/.cookie
2018-07-01T07:01:15Z HTTP: starting 4 worker threads
2018-07-01T07:01:15Z Using wallet directory /media/bitcoin/bitcoin/
2018-07-01T07:01:15Z init message: Verifying wallet(s)...
2018-07-01T07:01:15Z Using BerkeleyDB version Berkeley DB 4.8.30: (April  9, 2010)
2018-07-01T07:01:15Z Using wallet wallet.dat
2018-07-01T07:01:15Z BerkeleyEnvironment::Open: LogDir=/media/bitcoin/bitcoin/database ErrorFile=/media/bitcoin/bitcoin/db.log
2018-07-01T07:01:15Z Cache configuration:
2018-07-01T07:01:15Z * Using 2.0MiB for block index database
2018-07-01T07:01:15Z * Using 8.0MiB for chain state database
2018-07-01T07:01:15Z * Using 440.0MiB for in-memory UTXO set (plus up to 286.1MiB of unused mempool space)
2018-07-01T07:01:15Z init message: Loading block index...
2018-07-01T07:01:15Z Opening LevelDB in /media/bitcoin/bitcoin/blocks/index
2018-07-01T07:01:15Z Opened LevelDB successfully
2018-07-01T07:01:15Z Using obfuscation key for /media/bitcoin/bitcoin/blocks/index: 0000000000000000
2018-07-01T07:01:15Z LoadBlockIndexDB: last block file = 0
2018-07-01T07:01:15Z LoadBlockIndexDB: last block file info: CBlockFileInfo(blocks=0, size=0, heights=0...0, time=1970-01-01...1970-01-01)
2018-07-01T07:01:15Z Checking all blk files are present...
2018-07-01T07:01:15Z Initializing databases...
2018-07-01T07:01:15Z Pre-allocating up to position 0x1000000 in blk00000.dat
2018-07-01T07:01:15Z Opening LevelDB in /media/bitcoin/bitcoin/chainstate
2018-07-01T07:01:15Z Opened LevelDB successfully
2018-07-01T07:01:15Z Wrote new obfuscate key for /media/bitcoin/bitcoin/chainstate: d1fccc14bd785b5f
2018-07-01T07:01:15Z Using obfuscation key for /media/bitcoin/bitcoin/chainstate: d1fccc14bd785b5f
2018-07-01T07:01:15Z init message: Rewinding blocks...
2018-07-01T07:01:15Z  block index              10ms
2018-07-01T07:01:15Z init message: Loading wallet...
2018-07-01T07:01:15Z nFileVersion = 169900
2018-07-01T07:01:15Z Keys: 0 plaintext, 0 encrypted, 0 w/ metadata, 0 total. Unknown wallet records: 0
2018-07-01T07:01:15Z Performing wallet upgrade to 169900
2018-07-01T07:01:15Z keypool added 2000 keys (1000 internal), size=2000 (1000 internal)
2018-07-01T07:01:15Z  wallet                  850ms
2018-07-01T07:01:15Z setKeyPool.size() = 2000
2018-07-01T07:01:15Z mapWallet.size() = 0
2018-07-01T07:01:15Z mapAddressBook.size() = 0
2018-07-01T07:01:15Z UpdateTip: new best=000000000019d6689c085ae165831e934ff763ae46a2a6c172b3f1b60a8ce26f height=0 version=0x00000001 log2_work=32.000022 tx=1 date='2009-01-03T18:15:05Z' progress=0.000000 cache=0.0MiB(0txo)
2018-07-01T07:01:15Z mapBlockIndex.size() = 1

I also saw few articles stating that 0.16.99 is a dev version only and once there is a need to switch to other stable version -> I need to re-install bitcoin server. Any thoughts?

Tnx!
